src/home/sorting/sorting-manager.ts

```diff
@@ -55,17 +59,48 @@ export class SortingManager {
       try {
         const filterText = textBox.value.toLowerCase();
         const issues = Array.from(issuesContainer.children) as HTMLDivElement[];
+
+        // Reset any active sort buttons when searching
+        if (filterText) {
+          if (this) {
+            this._resetSortButtons();
```

Comment:
`this` isn't valid here because of scope, if you make filterIssues an arrow function it will work:

change from `function filterIssues() {` to `const filterIssues = () => {`